components/navigation/JobPostItem.js

```diff
@@ -1,9 +1,11 @@
 import { join } from "lodash";
+import Link from 'next/link'
 
 export default function JobPostItem({ job }) {
   return (
     <li>
-      <a href={job.slug} className="block hover:bg-gray-50">
+     <Link href={job.slug}>
```

Comment:
I wonder if it's needed, because `job.slug` should point to the external job item and `<Link/>` component for me is useless.

Reply:
Probably, when we will add a job to the website, a route will be made inside the website only, using slug. If we are using md to add a job, it's probably by slug to handle it on the website. Also, if it is an external link, the `Link` component will not make any difference with `a` tag there.

Reply:
You're right. Actually, I didn't think of it that way. Thanks!